src/locales/en/common.json

```diff
@@ -160,16 +182,17 @@
   "logout": "logout",
   "my votes": "my votes",
   "send": "send",
+  "separator": "separator",
   "your passphrase will be required for logging in to your account.": "your passphrase will be required for logging in to your account.",
   "your second passphrase will be required for all transactions sent from this account": "your second passphrase will be required for all transactions sent from this account",
-  "{{count}} delegate names successfully resolved to add vote to.": "{{count}} delegate name successfully resolved to add vote to.",
-  "{{count}} delegate names successfully resolved to add vote to._plural": "{{count}} delegate names successfully resolved to add vote to.",
-  "{{count}} delegate names successfully resolved to remove vote from.": "{{count}} delegate name successfully resolved to remove vote from.",
-  "{{count}} delegate names successfully resolved to remove vote from._plural": "{{count}} delegate names successfully resolved to remove vote from.",
+  "{{count}} delegate names successfully resolved to add vote to.": "{{count}} delegate names successfully resolved to add vote to.",
+  "{{count}} delegate names successfully resolved to add vote to._plural": "",
+  "{{count}} delegate names successfully resolved to remove vote from.": "{{count}} delegate names successfully resolved to remove vote from.",
+  "{{count}} delegate names successfully resolved to remove vote from._plural": "",
   "{{count}} of delegate names selected for unvote were not voted for:": "{{count}} of delegate names selected for unvote were not voted for:",
-  "{{count}} of delegate names selected for unvote were not voted for:_plural": "{{count}} of delegate names selected for unvote were not voted for:",
+  "{{count}} of delegate names selected for unvote were not voted for:_plural": "",
   "{{count}} of delegate names selected for vote were already voted for:": "{{count}} of delegate names selected for vote were already voted for:",
-  "{{count}} of delegate names selected for vote were already voted for:_plural": "{{count}} of delegate names selected for vote were already voted for:",
+  "{{count}} of delegate names selected for vote were already voted for:_plural": "",
   "{{count}} of entered delegate names could not be resolved:": "{{count}} of entered delegate names could not be resolved:",
-  "{{count}} of entered delegate names could not be resolved:_plural": "{{count}} of entered delegate names could not be resolved:"
+  "{{count}} of entered delegate names could not be resolved:_plural": ""
```

Comment:
Please revert changes on lines 149-197. There was a problem in `npm run i18n-scanner` that caused those changes, but now it should keep the strings intact.